xformers/components/attention/csrc/cuda/mem_eff_attention/kernel_backward.h

```diff
@@ -503,17 +503,6 @@ struct AttentionBackwardKernel {
     SharedStorage& shared_storage = *((SharedStorage*)smem_buffer);
     int32_t thread_id = threadIdx.x + threadIdx.y * blockDim.x;
 
-    auto clearSmem = [&]() {
-      // Initialize shared-memory. It can contain `nans` otherwise that screw up
-      // everything (only seens on Sm75+ tho)
-      uint32_t* smem = (uint32_t*)smem_buffer;
-      for (int i = 0; i < sizeof(SharedStorage) / sizeof(uint32_t) -
-               kWarpSize * kNumWarpsPerBlock;
-           i += kWarpSize * kNumWarpsPerBlock) {
-        smem[i + thread_id] = 0;
-      }
-    };
-
```

Comment:
no longer needed as we changed how we iterate on residuals - at least the tests pass without